geodetic_utils/src/gps_to_pose_conversion_node.cpp

```diff
@@ -110,6 +119,17 @@ int main(int argc, char** argv)
     g_is_sim = false;
   }
 
+  // Specify if covariances should be set manually or from GPS
+  ros::param::param("~trust_gps", trust_gps, false);
+
+  // Get manual parameters
+  ros::param::param("~manual_covariances/position/x", covariance_position_x, 5.0);
```

Comment:
Can somebody please check whether this is the standard way we read params with default values throughout the MAV code at least? @helenol @burrimi @ffurrer